src/utils/checkpointLoader.js

```diff
@@ -10,61 +10,39 @@ export default class CheckpointLoader {
     }
   }
 
-  loadManifest() {
-    return new Promise((resolve) => {
-      const xhr = new XMLHttpRequest();
-      xhr.open('GET', this.urlPath + MANIFEST_FILE);
-      xhr.onload = () => {
-        this.checkpointManifest = JSON.parse(xhr.responseText);
-        resolve();
-      };
-      xhr.onerror = (error) => {
-        throw new Error(`${MANIFEST_FILE} not found at ${this.urlPath}. ${error}`);
-      };
-      xhr.send();
-    });
+  async loadManifest() {
+    try {
+      this.checkpointManifest = await (await fetch(this.urlPath + MANIFEST_FILE)).json();
```

Comment:
possibly there was some reason why this bit of code used XMLHttpRequest? It's not apparent in the source before, and I think it'd be ideal if `fetch` is used uniformly.

Reply:
This was probably taken from the old checkpoint loader in deeplearnjs, there's likely no reason not to use fetch.

Reply:
yes, we took this from the previous checkpoint loader. thanks for the update.